release/pipelines.snapshot.yml

```diff
@@ -46,7 +46,7 @@ pipelines:
               JFROG_CLI_BUILD_NAME=$JFROG_CLI_BUILD_NAME
               JFROG_CLI_BUILD_NUMBER=$JFROG_CLI_BUILD_NUMBER
               JFROG_CLI_BUILD_PROJECT=$JFROG_CLI_BUILD_PROJECT
-              jf gradle clean build -x test artifactoryPublish
+              jf gradle clean build -x test
```

Comment:
```suggestion
              jf gradle clean build -x test artifactoryPublish
```

Reply:
We removed the aP task, shouldn't we remove it?

Reply:
We didn't. The JFrog CLI injects the Gradle Artifactory plugin with the artifactoryPublish task.
Without it, the jars will not be published to our server.